articles/getting-started/getting-started-sparkpost-enterprise.md

```diff
@@ -34,7 +34,7 @@ Examples:
     * Example: "binding" : "marketing"
 * Sizing considerations for multiple recipient transmissions:
     * ​5,000 - 10,000 recipients per transmission call is the recommended range for maximizing performance and efficiency.
-    * There is a 2GB limit on payload size for each transmission, but we recommend keeping a payload size no greater than 1GB.
+    * There is a 300MB limit on payload size for each transmission, but we recommend keeping a payload size no greater than 50MB.
```

Comment:
I thought that recommending a 50MB payload size was reasonable. In https://www.sparkpost.com/docs/tech-resources/smtp-rest-api-performance/, we also recommend 50MB.

Reply:
I'm good with that!